app/services/katello/ui_notifications/subscriptions/expire_soon.rb

```diff
@@ -0,0 +1,62 @@
+module Katello
+  module UINotifications
+    module Subscriptions
+      class ExpireSoon
+        class << self
+          def deliver!
+            subs_expiring_soon.each do |subscription|
+              next if notification_already_exists?(subscription) &&
+                update_expiration_date(subscription).zero?
+              ::Notification.create!(
+                :subject => subscription,
+                :initiator => User.anonymous_admin,
+                :audience => Notification::AUDIENCE_TAXONOMY,
```

Comment:
Does this require some update to foreman?  I'm getting an error that this constant isn't defined:

```
NameError: uninitialized constant Notification::AUDIENCE_TAXONOMY
	from /home/vagrant/git/foreman_hooks/lib/foreman_hooks/as_dependencies_hook.rb:4:in `load_missing_constant'
	from /home/vagrant/git/katello/app/services/katello/ui_notifications/subscriptions/expire_soon.rb:13:in `block in deliver!'
	from /home/vagrant/git/katello/app/services/katello/ui_notifications/subscriptions/expire_soon.rb:7:in `each'
	from /home/vagrant/git/katello/app/services/katello/ui_notifications/subscriptions/expire_soon.rb:7:in `deliver!'
```

Reply:
yeah, there's no such audience, I guess we need to find administrators of a given org for which there's no good option atm, @jlsherrill can we find a user who uploaded the manifest for the organization somehow?

Reply:
We don't track who has uploaded or refreshed a manifest.  I would suggest notifying anyone who has access to do that for the given org via the 'import_manifest' permission